common/app/views/fragments/atoms/quiz.scala.html

```diff
@@ -102,6 +103,26 @@
     }
 }
 
+@renderHeader(maybeResults: Option[QuizResults]) = {
+<div class="atom-quiz__end-message">
```

Comment:
maybe this should be `atom-quiz__response` or `atom-quiz__message`, or something similar, since it's not really at the end now?